tests/integration/transitions/transitions_core.js

```diff
@@ -97,12 +97,12 @@ NOTES:
 Our default transition handler now has either one or two animationComplete calls - two if there are two pages in play (from and to)
 To is required, so each async function must call start() onToComplete, not onFromComplete.
 */
-asyncTest( "changePage applies perspective class to mobile viewport for flip", function() {
+asyncTest( "change() applies perspective class to mobile viewport for flip", function() {
 	expect( 1 );
 
 	$.testHelper.pageSequence( [
 		function() {
-			$.mobile.changePage( "#foo" );
+			$( ".ui-pagecontainer" ).pagecontainer( "change", "#foo" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?